vector/src/main/java/im/vector/app/features/home/room/detail/composer/RichTextComposerLayout.kt

```diff
@@ -240,6 +240,12 @@ internal class RichTextComposerLayout @JvmOverloads constructor(
                 }
             }
         }
+        addRichTextMenuItem(R.drawable.ic_composer_bullet_list, R.string.rich_text_editor_bullet_list, ComposerAction.UNORDERED_LIST) {
+            views.richTextComposerEditText.toggleList(false)
```

Comment:
Maybe it would be more clear to use named parameter?
```suggestion
            views.richTextComposerEditText.toggleList(ordered = false)
```

Reply:
[`b762c53`](https://github.com/vector-im/element-android/pull/7887/commits/b762c53823e56e8f331a079762268e5094ca5800)